dotnet-dapper/ReturnInsertedIdentityWithDapper/ReturnInsertedIdentityWithDapper/SetUp.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Dapper;
+using Microsoft.Data.SqlClient;
+
+namespace ReturnInsertedIdentityWithDapper;
+
+public static class SetUp
+{
+    public static void CreateTable(string connectionString)
+    {
+        using var connection = new SqlConnection(connectionString);
+
+        connection.Open();
+
+        var createTableSql = @$"CREATE TABLE Students (Id INT IDENTITY(1,1) PRIMARY KEY, 
+            Surname VARCHAR(255), 
+            Firstname VARCHAR(255))";
```

Comment:
No need for this to be an interpolated string nor a verbatim string. The suggestion was for this to be a parameterized query using raw string literals.